data/pirate jobs.txt

```diff
@@ -2473,11 +2473,13 @@ mission "Capture Smuggler"
 		random < 15
 		"combat rating" > 2000
 		"reputation: Pirate" > 10
+	passengers 1
 	source
 		government "Pirate"
 	npc board
 		conversation
-			`After neutralizing the crew and capturing the captain, you sweep the ship. Ultimately you find the stolen cargo in the captain's cabin, hidden behind a false panel. The cargo is a nondescript storage container, but upon closer inspection you see that it's code-locked and covered with interference plating.`
+			`The smuggler and his crew are ready and waiting when you board. You pull out the "gift" and press the button, then quickly toss it down the hallway. For a moment, you meet the smuggler's eyes. They widen with terror when he sees the "gift". You duck back into cover. A few seconds later, you hear a strangly muted detonation, followed by silence. After a minute, you hazard another peek out of cover. The smuggler and all his men are unconscious on the ground. One crewman has blood dripping from his nose and ears. You tie them all up and stow the smuggler in a locked cabin on your ship.`
```

Comment:
The period after "gift" should be inside the quotation mark, since we're working in American English.